managed/services/scheduler/task.go

```diff
@@ -151,8 +151,8 @@ func NewMongoDBBackupTask(params *BackupTaskParams) (Task, error) {
 		return nil, errors.Errorf("unsupported backup mode for mongoDB: %s", params.Mode)
 	}
 
-	if params.DataModel != models.LogicalDataModel {
-		return nil, errors.Errorf("unsupported backup data model for mongoDB: %s", params.DataModel)
+	if params.Mode == models.PITR && params.DataModel != models.LogicalDataModel {
+		return nil, errors.Wrap(backup.ErrIncompatibleDataModel, "PITR is only supported for logical backups")
```

Comment:
This usage of `ErrIncompatibleDataModel` looks especially strange. The original purpose of Wrap is to provide additional context to the original errors. In this PR we don't do that for all `ErrIncompatibleDataModel` usages.

Reply:
I don't get the "original error" part - We do a similar thing for the other errors (https://github.com/percona/pmm/blob/da7a40bd5658e9ef952ea035e035c2788ccc68ba/managed/services/backup/backup_service.go#L686), and we can compare the error with `errors.Is()`